pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -82,6 +87,10 @@ const (
 	imageDigestExporterContainerName = "step-image-digest-exporter"
 )
 
+var (
+	nopImage = flag.String("nop-image", "override-with-nop:latest", "The container image used to kill sidecars")
```

Comment:
can we update the nopImage `cmd` source to explain why we have this around still? (since we don't need to include it as an actual step now that we're not using init containers for steps)

Reply:
I've added a comment to the nop cmd explaining its reason for still existing.